containers/Reports/__tests__/ReportCreateMessage.test.js

```diff
@@ -1,71 +1,55 @@
 import React from 'react'
-import { fireEvent, render, screen } from '@testing-library/react-native'
+import { act, fireEvent, render, screen } from '@testing-library/react-native'
 
 import '@testing-library/jest-native/extend-expect'
 
 import { testIdeaAdmin, testText } from '../../../tests/TestData'
 import { ReportCreateMessage } from '../ReportCreateMessage'
 
-test('Test ReportCreateMessage Snapshot', () => {
-  const route = { params: {
-    content_type: testIdeaAdmin.content_type,
-    object_pk: testIdeaAdmin.pk
-  }}
-  const { toJSON } = render(<ReportCreateMessage route={route}/>)
+jest.mock('react-native/Libraries/Animated/NativeAnimatedHelper')
```

Comment:
is this to ensure the error messages are triggered?

Reply:
no, that's just to remove a warning which was in the terminal